### DISCUSSION TITLE
Select Multiple Items

### DISCUSSION THREAD

**Discussion by willtheorangeguy** [community/community#11198] [Projects and Issues]
Not sure if this is planned or not, but it would be really useful to be able to select multiple items to batch apply fields like assignee and status in the projects list view. It would speed up the workflow a lot. 

Maybe a checkbox could be added on the far left side?

Thanks!

**Comment by emiltin** [comment]
I also miss this.

**Reply by drone1** [reply]
Currently it seems you can select multiple in Board view only (by holding ctrl or shift while left-clicking). But not in Tasks view, where it makes a lot of sense also IMO.

**Reply by willtheorangeguy** [reply]
Ok, that is good to know. I guess I will just use it from there for now on.

**Reply by gradaom** [reply]
Awesome Idea, @drone1 !
